crates/p2p_proto/proto/sync.proto

```diff
@@ -28,42 +30,32 @@ enum Direction {
   BACKWARD = 1;
 }
 
+// Retrieve at most `count` block headers staring at `start_block`
```

Comment:
I left only one comment like this to make the convention wrt `count` and `size_limit` clear. I don't think we need redundant comments.